dashboard/app/controllers/lessons_controller.rb

```diff
@@ -8,7 +8,8 @@ def show
     @lesson_data = {
       unit: {
         displayName: @lesson.script.localized_title,
-        link: @lesson.script.link
+        link: @lesson.script.link,
+        lessons: @lesson.script.lessons.map {|lesson| {displayName: lesson.localized_name, link: lesson_path(id: lesson.id)}}
```

Comment:
I had it on my list to send more data down here next. I hadn't started on it yet, but let's coordinate to avoid merge conflicts. if you end up taking this, I'd recommend moving to some kind of `summarize` methods soon, not necessarily in this PR though because the list of stuff you're sending down right now seems short enough.

Reply:
I think this is the last information I need to send to the Lesson plan for my work this sprint. So if you want to take this and update after I merge that seems good to me